src/test/java/SignUp.java

```diff
@@ -1,80 +1,107 @@
-        driver.findElement(By.cssSelector("[placeholder='Password']")).sendKeys("testingPassword");
-        driver.findElement(By.cssSelector("[placeholder='Confirm Password']")).sendKeys("testingPassword");
-        driver.findElement(By.cssSelector("[placeholder='Zipcode']")).sendKeys("90020");
-
-        WebElement checkBoxOne = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[class='styled__CheckboxWrapper-sc-1x07d2p-2 cjrWjQ'] [class*='ws-checkbox']")));
-        checkBoxOne.click();
+    public static String getRandomEmail() {
+        Timestamp timestamp = new Timestamp(System.currentTimeMillis());
+        return timestamp.getTime() + "@gmail.com";
+    }
 
-        WebElement checkBoxTwo = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[class='styled__CheckboxWrapper-sc-1x07d2p-2 cIPcIz'] [class*='ws-checkbox']")));
-        checkBoxTwo.click();
+    public static void setEmailField(String email){
+        WebElement emailField = driver.findElement(By.cssSelector("[placeholder='Email']"));
+        emailField.sendKeys(email);
+    }
 
-        driver.findElement(By.xpath("//button[contains(text(),'Sign Up')]")).click();
+    public static void setWrongEmailField(String email){
+        WebElement emailField = driver.findElement(By.cssSelector("[placeholder='Email']"));
+        emailField.sendKeys(email);
+    }
 
-        WebElement errorAlert = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[content='Oops!']")));
-        Assert.assertTrue(errorAlert.isDisplayed());
+    public static void setPassword(String password){
+        WebElement passwordField = driver.findElement(By.cssSelector("[placeholder='Password']"));
+        passwordField.sendKeys(password);
     }
 
-    @Test
-    public static void signUpEmptyCredentials(){
-        driver.findElement(By.xpath("//div[@class='styled__Wrapper-sc-5elvgs-0 gTjIWz']/div/div[3]")).click();
+    public static void setConfirmPassword(String password){
+        WebElement passwordConfirm = driver.findElement(By.cssSelector("[placeholder='Confirm Password']"));
+        passwordConfirm.sendKeys(password);
+    }
 
-        driver.findElement(By.cssSelector("[placeholder='Email']")).sendKeys("");
-        driver.findElement(By.cssSelector("[placeholder='Password']")).sendKeys("");
-        driver.findElement(By.cssSelector("[placeholder='Confirm Password']")).sendKeys("");
-        driver.findElement(By.cssSelector("[placeholder='Zipcode']")).sendKeys("");
+    public static void setZipCode(String zipCode){
+        WebElement zip = driver.findElement(By.cssSelector("[placeholder='Zipcode']"));
+        zip.sendKeys(zipCode);
+    }
 
+    public static void clickCheckboxTermsAndConditions(){
         WebElement checkBoxOne = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[class='styled__CheckboxWrapper-sc-1x07d2p-2 cjrWjQ'] [class*='ws-checkbox']")));
```

Comment:
t would be better move this wait into the method with name "waitElementVisibility"and place it in "Base" class and then you can use it in other classes without duplicating wait.until for each element.